src/Layers/xrRenderPC_GL/xrRender_GL.cpp

```diff
@@ -29,8 +29,36 @@ XR_EXPORT bool CheckRendererSupport()
         return false;
     }
 
-    // Check if minimal required OpenGL features are available
-    // XXX: implement minimal feature availability check
+    SDL_Window* gl_test_window = SDL_CreateWindow("GL test", 0,0,1,1, SDL_WINDOW_HIDDEN|SDL_WINDOW_OPENGL);
+
+    if (!gl_test_window)
+    {
+        Msg("Cannot create helper window for OpenGL: %s", SDL_GetError());
+        return false;
+    }
+
+    SDL_GLContext gl_test_context = SDL_GL_CreateContext(gl_test_window);
+    if (!gl_test_context)
+    {
+        Msg("Cannot create OpenGL context: %s", SDL_GetError());
+        return false;
+    }
+
+    if (glewInit() != GLEW_OK)
+    {
+        Msg("Could not initialize glew.");
+        return false;
+    }
+
+    if (!glewIsSupported("GL_VERSION_4_1 GL_EXT_multi_draw_arrays"))
+    {
+        Msg("GL_VERSION_4_1 or GL_EXT_multi_draw_arrays not supported\n");
```

Comment:
Don't use Msg if there's no need to format message. Also, there's no need to type `\n`, it will be inserted automatically.
```suggestion
        Log("~ GL_VERSION_4_1 or GL_EXT_multi_draw_arrays not supported");
```